cloudprem/modules/webhooks/variables.tf

```diff
@@ -0,0 +1,49 @@
+variable "vpc_id" {
+  description = "The VPC ID where we'll be deploying our resources. (If creating a new VPC leave this field and subnets blank). When using an existing VPC be sure to tag at least 2 subnets with type = public and another 2 with tag type = private"
+  type        = string
+  default     = ""
+}
+variable "subnet_ids" {
+  type        = list(string)
+  default     = []
+  description = "AWS subnet ids"
+}
+variable "name" {
+  type = string
+  default = ""
+  description = "Name of the MSK cluster"
+}
+variable "rds_kms_key_id" {
+  type = string
+  default = ""
+  description = "KMS Key ID"
+}
+variable "cluster_size" {
+  type = number
+  default = 2
```

Comment:
Is `2` a good default here? Do these guys build an "on-cluster" zookeeper? `2` is probably a bad value for Raft..

We're using MSK for managed Kafka? Or is this run "bare metal" in the cluster?

Reply:
That variable value will actually never be used so it should be removed. The cluster size minimum for MSK is equal to the amount of AZ's in the VPC it's deployed into so that's what I set it to. In the future if that's not enough I'll expose the variable to the customer config so they can increase it.